### ISSUE TITLE
GCM, iOS, and Background notifications issue.

### ISSUE THREAD

**Issue by Moghul** [phonegap/phonegap-plugin-push#1545]
### Expected Behaviour
As described [here](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#background-notifications-1), you're supposed to be able to add "content-available" : 1 to the payload, and the on('notification') event should trigger. Then, you should get 30s of runtime.

I've added this key-value pair for a GCM payload pretty much everywhere, and varied it with every combination of 1, "1", true, "content_available", "content-available", "contentAvailable" that I could think of.
### Actual Behaviour

The event does not trigger, and none of the NSLogs in the plugin trigger either. Tapping the notification, however, does trigger the NSLogs, and the event, of course.

### Reproduce Scenario (including but not limited to)

Using GCM, I sent my notification, and looked for the logs from my code, and from the plugin code, which never triggered. The app was killed (not in the carousel thing) the whole time, until I tapped the notification.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)

iOS 10.2

#### Cordova CLI version and cordova platform version

    cordova ios version 4.3.1

#### Plugin version

    phonegap-plugin-push 1.9.1

#### Sample Push Data Payload
```
{
    "priority" : "high",
    "notification" : {
        "title": "Test Notification",
        "body": "Please disregard"
    },
    "data" : {
        "content-available" : "1",
        "notId": 10,
    	"someData": "myAdditionalData"
    },
    "to" : "my gcm id"
}
```
#### Sample Code that illustrates the problem

It's simply that the code inside the two didReceiveRemoteNotification functions in the AppDelegat+notification.m does not trigger until the notification is tapped.

#### Logs taken while reproducing problem
N/A

**Comment by jcesarmobile** [comment]
content-available has to be on the root of the aps object, not sure how you can do it with GCM, but it's not a problem with the plugin

Example payload that you can use:
```
{
    aps =     {
        "content-available" = 1;
        sound = "";
    };
}
```

According to this, you have to use content_available
https://developers.google.com/cloud-messaging/http-server-ref

**Comment by Moghul** [comment]
Thanks for the reply. As for the actual answer, if this is happening to anyone else, please check this post, and see if the log has this message.

https://forums.developer.apple.com/thread/31403

TL;DR content_available doesn't work if the app is force-killed. I would love to see this as a note in the docs.
